dist/AjaxDynamicDataTable.js

```diff
@@ -0,0 +1,78 @@
+function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DynamicDataTable from "./DynamicDataTable";
+import { AxiosInstance as axios } from "axios";
+
+class AjaxDynamicDataTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rows: [],
+      currentPage: 1,
+      totalPages: 1,
+      orderByField: null,
+      orderByDirection: null
+    };
+  }
+
+  componentDidMount() {
+    this.loadPage(1);
+  }
+
+  render() {
+    const state = this.state;
```

Comment:
Please use object destructuring to access objects in the state

Reply:
Done.